variables.tf

```diff
@@ -10,6 +10,12 @@ variable "security_group" {
   description = "Security group to allow load balancer traffic to"
 }
 
+variable "sc_security_group" {
+  description = "State channels security group to allow load balancer traffic to"
+  type        = string
+  default     = "unknown"
```

Comment:
no need for "unknown", "" should do - isn't it?

Reply:
d1a7793